repo/actions/registry.go

```diff
@@ -42,6 +42,21 @@ func (act Registry) Unpublish(ref repo.DatasetRef) (err error) {
 	return cli.DeleteDataset(ref.Peername, ref.Name, ds.Encode(), pub)
 }
 
+// Status checks to see if a dataset is published to a repo's specific registry
+func (act Registry) Status(ref repo.DatasetRef) (err error) {
+	cli, _, _, err := act.dsParams(&ref)
+	if err != nil {
+		return err
+	}
+	if err = act.permission(ref); err != nil {
```

Comment:
the "permission" check should be dropped. "permission" checks to see if this peer owns the dataset in question, but peers should be able to check the status of datasets they don't own.